Bottom: de32e13e330a570a739d2297411e4ea22fa5fe0b
Top:    6cb996f496262d47e25b946da1a5e6b53669e44a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 13:41:07 +0000

Refresh of play-with-quad-edge-data-STEP

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 56114d9..fec88cf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -243,6 +243,7 @@ get_contour_coord_n_in_mm (PLINE *contour, int n, double *x, double *y)
   *y = COORD_TO_MM (vertex->point[1]); /* FIXME: PCB's coordinate system has y increasing downwards */
 }
 
+#if 0
 void
 object3d_export_to_step (object3d *object, const char *filename)
 {
@@ -797,6 +798,7 @@ quad_emit_board_contour_step (FILE *f, PLINE *contour)
 
   fclose (f);
 }
+#endif
 
 object3d *
 object3d_from_board_outline (void)
diff --git a/src/hid/common/object3d.h b/src/hid/common/object3d.h
index b967543..5cb6ca3 100644
--- a/src/hid/common/object3d.h
+++ b/src/hid/common/object3d.h
@@ -9,5 +9,5 @@ object3d *make_object3d (void);
 void destroy_object3d (object3d *object);
 object3d *object3d_create_test_cube (void);
 object3d *object3d_from_board_outline (void);
-void object3d_export_to_step (object3d *object, const char *filename);
+//void object3d_export_to_step (object3d *object, const char *filename);
 void object3d_test_board_outline (void);
